test/token/ERC721/ERC721.behavior.js

```diff
@@ -173,7 +173,7 @@ function shouldBehaveLikeERC721 (
             const tokensListed = await Promise.all(
               [0, 1].map(i => this.token.tokenOfOwnerByIndex(owner, i))
             );
-            tokensListed.map(t => t.toNumber()).should.have.members([firstTokenId, secondTokenId]);
+            tokensListed.should.have.deep.members([firstTokenId, secondTokenId]);
```

Comment:
This works but is not ideal IMO because deep equality is not necessarily the same as bignumber equality. I've opened https://github.com/ZeppelinSolutions/chai-bn/issues/5 to discuss a better solution.